src/App.vue

```diff
@@ -10,7 +10,8 @@ import { type DrugProp } from '@/Drug.type';
 const resultRef = ref<{ result: DrugProp[] }>({ result: [] });
 const searchText = ref("");
 
-const searchDrugs = (drugSearch: string) => {
+const searchDrugs = async (drugSearch: string) => {
```

Comment:
While using the app, I am running into some very jagged behavior- When filling out the search bar, the app is freezing, especially when removing what I've typed into the input.

I appreciate the thought behind designing this app the way you have, but the search triggering a fetch each time the input updates isn't working the way we want it. I think scaling back to only searching when the submit button is clicked would improve behavior a bunch.